src/safeds/exceptions/__init__.py

```diff
@@ -58,6 +59,7 @@
     "ModelNotFittedError",
     "PredictionError",
     "UntaggedTableError",
+    "NonTimeSeriesError",
```

Comment:
[Sort this alphabetically](https://library.safeds.com/en/latest/development/project_guidelines/#sort-exported-classes-in-__init__py)

Reply:
changed